recipes/libcuba/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "4.2.2" %}
+
+package:
+  name: libcuba
+  version: {{ version }}
+
+source:
+  # url: https://feynarts.de/cuba/Cuba-{{ version }}.tar.gz
+  # sha256: 8d9f532fd2b9561da2272c156ef7be5f3960953e4519c638759f1b52fe03ed52
+
+  # upstream did not respond yet to submitted patches
+  git_rev: "main"
+  git_url: "https://github.com/jschueller/cuba.git"
```

Comment:
Do not use a tag from your own fork. There is no easy way for someone to audit how your fork has diverged from the canonical repository.

It's inconvenient for large patches, but text formatted patch included with this recipe is best.

Reply:
ok, done